Route.html

```diff
@@ -9,6 +9,8 @@
 <script>
 import { getHistory, matchPath } from './index.js';
 
+const defaultData = ['match', 'pathname', 'path', 'exact', 'strict', 'component', 'props', 'history'];
```

Comment:
These are all the props and data attributes that we don't want to pass to the child component.